spec/forms/openid_connect_authorize_form_spec.rb

```diff
@@ -144,9 +144,16 @@
       end
     end
 
-    context 'without the optional nonce' do
-      let(:nonce) { nil }
-      it { expect(valid?).to eq(true) }
+    context 'nonce' do
+      context 'without a nonce' do
+        let(:nonce) { nil }
+        it { expect(valid?).to eq(false) }
+      end
+
+      context 'with a nonce that is shorter than 32 characters' do
+        let(:nonce) { '123456' }
```

Comment:
Personally I think it's easier to just see a value that's shorter? How strongly do you feel about this style?